### ISSUE TITLE
GCPBigQueryDatasetWorldReadableConstraintV1 false positive

### ISSUE THREAD

**Issue by rosmo** [forseti-security/policy-library#357]
I'm getting lots of warnings about BigQuery datasets being public. Example:

```
- //bigquery.googleapis.com/projects/foo-dev-eng-customer-nonprod/datasets/TestLogs is publicly accessable
```
IAM policies for the dataset:
```
{
    "ancestors": [
        "projects/X",
        "folders/Y",
        "folders/Z",
        "organizations/FOO"
    ],
    "asset_type": "bigquery.googleapis.com/Dataset",
    "iam_policy": {
        "bindings": [
            {
                "members": [
                    "projectEditor:foo-dev-eng-customer-nonprod"
                ],
                "role": "roles/bigquery.dataEditor"
            },
            {
                "members": [
                    "projectOwner:foo-dev-eng-customer-nonprod",
                    "user:user@customer.com"
                ],
                "role": "roles/bigquery.dataOwner"
            },
            {
                "members": [
                    "projectViewer:foo-dev-eng-customer-nonprod"
                ],
                "role": "roles/bigquery.dataViewer"
            }
        ],
        "etag": "BwWbfi+ZbgU="
    },
    "name": "//bigquery.googleapis.com/projects/foo-dev-eng-customer-nonprod/datasets/TestLogs"
}
```

There's no allUsers or allAuthenticatedUsers anywhere.

**Comment by morgante** [comment]
What client are you using? Looking at the rule, I don't see any obvious reason for this failing: https://github.com/forseti-security/policy-library/blob/4e070db149c2f969400df22804baa63c0c5c64dc/validator/bigquery_dataset_world_readable.rego

**Comment by rosmo** [comment]
Ah, sorry. I'm using CFT Scorecard.

**Comment by morgante** [comment]
I'm able to replicate this, working on a fix.

**Comment by morgante** [comment]
Turns out we'd forgotten to sync the policy, it's a trivial fix - #358
